src/protocols/two_party_ecdsa/lindell_2017/party_two.rs

```diff
@@ -233,7 +237,7 @@ impl Party2Private {
     // used for verifiable recovery
     pub fn to_encrypted_segment(
         &self,
-        segment_size: &usize,
+        segment_size: usize,
```

Comment:
here as well. Breaks Backward compatibility. Is it critical ?

Reply:
it's okay to break it since we are at beta stage

Reply:
what beta stage ?

Reply:
https://github.com/KZen-networks/multi-party-ecdsa/blob/master/Cargo.toml#L3

semver

Reply:
right right